app.go

```diff
@@ -197,6 +210,16 @@ func (a *App) RunAsSubcommand(ctx *Context) error {
 		}
 	}
 
+	if a.After != nil {
+		defer func() {
+			aferr := a.After(context)
+			// check Action error to avoid shadowing
+			if err == nil {
```

Comment:
I don't believe `err` will ever be not `nil` here. The only time it is set is on line 178 above and there is a check on 193 to see if it is non-nil.